components/org.wso2.carbon.identity.api.user.association/org.wso2.carbon.identity.api.user.association.v1/src/main/resources/association.yaml

```diff
@@ -126,13 +126,24 @@ paths:
              $ref: '#/definitions/Error'
 
     delete:
-      summary: Delete all my user associations
+      summary: Delete my user associations
       description: |
-        This API is used to delete all associations of the auhtentiated user.
+        This API is used to delete a given association(s) of the auhtentiated user.
```

Comment:
Change the description to mention that all the associations will be removed if query params are not defined